### PR TITLE
[risk=low][no ticket] Some module UI refactoring ideas

### PR DESCRIPTION
**Pull request by jmthibault79**
A few things I thought of after looking at #8363 

Tested by viewing the DAR and AAR pages

---
**PR checklist**

- [x] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have manually tested this change and my testing process is described above.
- [x] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [x] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [x] None of the above apply to this change.

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/config/WorkbenchConfigMapper.java [hunk 1]
```diff
@@ -40,6 +40,8 @@ default void mapRuntimeImages(WorkbenchConfig source, @MappingTarget ConfigRespo
 
   // handled by mapRuntimeImages()
   @Mapping(target = "runtimeImages", ignore = true)
+  // deprecated, for removal
+  @Mapping(target = "redirectMoodleToAbsorb", constant = "true")
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/config/WorkbenchConfigMapper.java:44]: This is now always true, so we can hard-code it in preparation for removal

File: ui/src/app/pages/access/module.tsx [hunk 2]
```diff
@@ -81,11 +81,6 @@ export const Module = (props: ModuleProps) => {
   const { refreshAction, isEnabledInEnvironment } =
     getAccessModuleConfig(moduleName);
 
-  const enableTrainingComponent = () => {
```
Comment (jmthibault79) [inline ui/src/app/pages/access/module.tsx:84]: this is a generic action for all modules, so naming it "training" is confusing

File: ui/src/app/pages/access/module.tsx [hunk 3]
```diff
@@ -106,10 +101,14 @@ export const Module = (props: ModuleProps) => {
           !(enableRasIdMeLinking && moduleName === AccessModule.IDENTITY)
         }
         action={() => {
+          // disable action to prevent double-clicking
```
Comment (jmthibault79) [inline ui/src/app/pages/access/module.tsx:104]: describe why we are doing this

**Review by NehaBroad — APPROVED** [review]
Thank you!

**Pull request merged**